public/index.html

```diff
@@ -369,6 +369,63 @@ <h1 style="background-color: {{grid.headingcolor}} !important;">{{ grid.header }
                     <div class="info">Best of {{tennisOptions.maxSets}} sets</div>
                 </div>
             </div>
+			
+			<div class="tennis tennisStats center scale-fade" ng-show="tennisOptions.showStats">
+			   	    <div class="header">
+					    <h1>Match Statistics</h1>
+				    </div>
+					<div class="info" style="width: 100%">{{tennisOptions.matchName}} - After {{tennisScore.sets1.length}} of {{tennisOptions.maxSets}} sets</div>
+
+					<div class="row">
+						<div class="stats names">
+						    <span class="set" ng-repeat="i in tennisScore.sets1 track by $index">{{tennisScore.sets1[$index] + "-" + tennisScore.sets2[$index]}}</span>
+							<span style="float:left;">{{tennisOptions.player1}}</span>
```

Comment:
Inline floats seem a bit odd given we are using classes for left and right alignment in other places

Reply:
Fixed.